sdk/keyvault/keyvault-certificates/package.json

```diff
@@ -20,15 +20,16 @@
     "url": "https://github.com/Azure/azure-sdk-for-js/issues"
   },
   "main": "./dist/index.js",
-  "module": "dist-esm/src/index.js",
+  "module": "dist-esm/keyvault-certificates/src/index.js",
```

Comment:
why?

Reply:
After linking the external folder, TypeScript creates two folders in the `dist-esm` target directory, one for `keyvault-certificates` and another one for `keyvault-common`.